src/applications/check-in/components/pages/AppointmentDetails/index.jsx

```diff
@@ -219,6 +230,20 @@ const AppointmentDetails = props => {
                     />
                   </div>
                 )}
+              {isPreCheckIn &&
+              !isUpcoming &&
+              pages.length > 4 && ( // pre-check-in is not completed
```

Comment:
I don't think pages length is a good indicator of if pre-check-in is complete. I would use the sessionStorage `health.care.pre.check.in.complete` value here instead

Reply:
not instead but in addition to. Because the details link is visible after completion